src/main/scala/com/research/nomad/markii/GUIAnalysis.scala

```diff
@@ -128,6 +128,16 @@ object GUIAnalysis extends IAnalysis {
     printWriter.close()
   }
 
+  private def getJavaLineNumber(sootMethod:SootMethod):Int= {
```

Comment:
nit: try to follow the style of existing code

Reply:
Can you be more specific on this comment?

Reply:
```scala
  private def getJavaLineNumber(sootMethod: SootMethod): Int = {
```

would be better

Reply:
No problem